src/app/services/dialog.js

```diff
@@ -1,13 +1,34 @@
+/**
+ * This factory exposes methods for showing custom dialogs, alerts and teasers.
+ *
+ * @module app
+ * @submodule dialog
+ */
 app.factory('dialog', ($mdDialog, $mdToast, $mdMedia) => ({
 
+  /**
+   * Uses mdDialog to show a toast with error theme
+   *
+   * @param {String} text - The message of the toast
+   * @returns {promise} The mdDialog promise
+   */
   errorToast(text) {
     return this.toast({ success: false, text });
   },
 
+  /**
+   * Uses mdDialog to show a toast with success theme
+   *
+   * @param {String} text - The message of the toast
+   * @returns {promise} The mdDialog promise
+   */
   successToast(text) {
     return this.toast({ success: true, text });
   },
 
+  /**
+   * @private
```

Comment:
So far we're not using this publicly, but it makes sense to use it publicly. Same for alert.